### PR TITLE
fix: fixed utc problems in RecurrenceWidget

### PR DESCRIPTION
**Pull request by giuliaghisini**
Fixed utc errors in RecurrenceWidget for:

- default until date
- recurrence start date

### CODE REVIEW

File: src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx
Comment (pnicolli) [inline src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx:473]: Checking `formData?` with the `?` is not needed because it is already checked in the previous line.

**Review by pnicolli — CHANGES_REQUESTED** [review]
Is the function `getUTCDate` not used anymore with these changes? If it is not, I would also remove that. I am currently on my phone, so I don't see it easily.

**Comment by giuliaghisini** [comment]
> 

the getUTCDate is used in componentDidUpdate

**Review by pnicolli — APPROVED** [review]

File: .env.local
Comment (pnicolli) [inline .env.local:2]: Remove this whole file. Use the `.env` name without `.local` when developing core volto.

**Review by pnicolli — CHANGES_REQUESTED** [review]

**Review by sneridagh — APPROVED** [review]
Sorry, this got buried in the PR avalanche. Is it ready?

**Comment by giuliaghisini** [comment]
> Sorry, this got buried in the PR avalanche. Is it ready?

yes

**Comment by stevepiercy** [comment]
@giuliaghisini would you please resolve the merge conflicts?

**Comment by davisagli** [comment]
@sneridagh I'm confused, this was merged for 17.x (#5555) but not for main?

File: packages/volto/news/5002.bugfix
Comment (stevepiercy) [inline packages/volto/news/5002.bugfix:1]: Exactly what problems are fixed?

Give yourself credit!

```suggestion
Fixed UTC problems in `RecurrenceWidget`. @giuliaghisini
```

File: packages/volto/src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx
Comment (stevepiercy) [inline packages/volto/src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx:226]: Commented code is evil code.

```suggestion
```

File: packages/volto/src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx
Comment (stevepiercy) [inline packages/volto/src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx:266]: This is more concise.

```suggestion
         // The `start` date from Plone is in UTC
        const _start = new Date(start);
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Just suggestions on comments and news items. This still needs review from a core contributor.

**Comment by stevepiercy** [comment]
Also per @davisagli's comment, the suggestions should be applied to the backport of this PR in https://github.com/plone/volto/pull/5555

**Comment by sneridagh** [comment]
@giuliaghisini what's the status of this one? Did we merge it in 17.x but not in main? Can we push forward this one?

**Comment by stevepiercy** [comment]
@giuliaghisini there's a merge conflict that needs to be resolved. I'm not sure what to do with this.

**Comment by giuliaghisini** [comment]
> @giuliaghisini there's a merge conflict that needs to be resolved. I'm not sure what to do with this.

resolved

**Comment by stevepiercy** [comment]
@giuliaghisini can you run lint and push? See https://6.docs.plone.org/volto/contributing/linting.html, and if the instructions are not correct, would you please let me know what needs correction so I can update them? Thank you!

**Comment by giuliaghisini** [comment]
> https://6.docs.plone.org/volto/contributing/linting.html

thanks.
Done.

this command
"pnpm lint:fix"
mentioned in the docs doesn't exists. Maybe docs or package.json commands needs to be updated

**Comment by stevepiercy** [comment]
Do you have pre-commit installed? I think that that will automatically run commands for you when you stage a commit. Otherwise you can look in `.lintstagedrc` to find the command:

`pnpm --filter @plone/volto lint:husky`

I have not been keeping up with the rapid changes. @sneridagh can you help out with this?

@giuliaghisini meanwhile, would you please add a [Change log entry](https://6.docs.plone.org/contributing/index.html#change-log-entry)? 

I'm also not sure why the acceptance tests failed. I restarted them.

**Pull request merged**
